Tests/Unit/MSFT_WindowsPackageCab.Tests.ps1

```diff
@@ -101,6 +104,14 @@ try
                     Set-TargetResource -Name $script:testPackageName -SourcePath $script:testSourcePath -Ensure 'Absent' -LogPath $script:testLogPath
                     Assert-MockCalled -CommandName 'Dism\Remove-WindowsPackage' -ParameterFilter { $LogPath -eq $script:testLogPath }
                 }
+                It 'Should not throw when a log path is not specified to Remove-WindowsPackage' {
```

Comment:
How are these two new tests any different from the two tests above?
If this scenario was failing wouldn't we see these tests failing already?
The parameter filter can be added to the tests above rather than adding new tests.
```
It 'Should call Add-WindowsPackage when Ensure is Present' {
     Set-TargetResource -Name $script:testPackageName -SourcePath $script:testSourcePath -Ensure 'Present'
     Assert-MockCalled -CommandName 'Dism\Add-WindowsPackage'
}
 	 
It 'Should call Remove-WindowsPackage when Ensure is Absent' {
    Set-TargetResource -Name $script:testPackageName -SourcePath $script:testSourcePath -Ensure 'Absent'
    Assert-MockCalled -CommandName 'Dism\Remove-WindowsPackage
}
```